index.js

```diff
@@ -0,0 +1,63 @@
+const title = document.getElementById('Title');
+const author = document.getElementById('Author');
+const addButton = document.getElementById('AddButton');
+const booksSection = document.getElementById('Books-section');
+
+const keys = Object.keys(localStorage);
+if (localStorage.length > 0) {
+  keys.forEach((key) => {
+    const newTitle = document.createElement('p');
+    newTitle.textContent = JSON.parse(localStorage.getItem(key)).Title;
+    const newAuthor = document.createElement('p');
+    newAuthor.textContent = JSON.parse(localStorage.getItem(key)).Author;
```

Comment:
I see you have tried a path to using `LocalStorage`, but the approach is not viable and You are not accessing Local Storage
![image](https://github.com/sserwangaedrisa/Awesome-Books-Store/assets/42886828/5f489329-0c25-4230-8f9a-1bc78a6ccfe1)
From the Image above you can see that data is not being loaded into local Storage.
Internalize the approach below and work on the remaining;

 _check if the local storage is empty, then add empty array_
```
if (localStorage.getItem('Added books') == null) {
  localStorage.setItem('Added books', JSON.stringify([]));
}
```
That is an example on how to handle localStorage, that way you can make functions referencing it by key